addon/components/frost-table-cell.js

```diff
@@ -34,5 +36,10 @@ export default Component.extend({
   // == Actions ===============================================================
 
   actions: {
+    handleCallback (...args) {
```

Comment:
I was worried that having an optional `onCallback` would break it when I tried to use it in a template; making the `onCallback` field required in the row/cell components as you suggested would definitely simplify this.

I had shied away from doing that as I didn't want to update the normal frost table yet.